lib/src/main/java/com/dynatrace/metric/util/Metric.java

```diff
@@ -302,6 +305,13 @@ public String serialize() throws MetricException {
         builder.append(time.toEpochMilli());
       }
 
+      if (builder.length() > METRIC_LINE_MAX_LENGTH) {
+        throw new MetricException(
+            String.format(
+                "Line exceeds threshold of %d characters and cannot be ingested into Dynatrace:\n%s",
```

Comment:
```suggestion
                "Serialized line exceeds limit of %d characters accepted by the ingest API:%n%s",
```
"cannot be ingested" sounds as if the library would've tried by itself